projects/burnr/src/hooks/api/useApi.ts

```diff
@@ -1,4 +1,5 @@
 import { useContext } from "react"
+import "@polkadot/api-augment"
```

Comment:
What's that and why is it now necessary?

Reply:
Based on polkadotJS [FAQ](https://polkadot.js.org/docs/api/FAQ/#since-upgrading-to-the-7x-series-typescript-augmentation-is-missing)  this is advised as the solution on the "breakings" that were appearing:
```
Since the start of the 7.x series, type augmentation is to be applied explicitly. To restore Substrate-only types as found earlier versions, an import '@polkadot/api-augment' is to be added to your codebase entry point before any imports from the API itself. The various shipped aliases available are the folowing -
```